Mathmagician.Tests/FibonacciTest.cs

```diff
@@ -52,6 +52,49 @@ public void FibonacciEnsureListCount()
 
         }
 
+
+        public int fibotest(int a, int b)
+        {
+
+            int temp = a;
+            a = b;
+            b = temp + b;
+            return b;
+        }
```

Comment:
It'll only be necessary in your Fibonacci.cs if you at any point need the method. Otherwise you can ditch the test and the method.